content/en/docs/tutorials/security/cluster-level-pss.md

```diff
@@ -59,18 +59,17 @@ that are most appropriate for your configuration, do the following:
     kubectl cluster-info --context kind-psa-wo-cluster-pss
     
     Thanks for using kind! 😊
-    
     ```
```

Comment:
The above paragraphs should be indented by 3 spaces.
4 spaces is bad because markdown may treat it as code snippets.